shared/patches/expo-image+1.8.0.patch

```diff
@@ -0,0 +1,74 @@
+diff --git a/node_modules/expo-image/android/src/main/java/expo/modules/image/ExpoImageViewWrapper.kt b/node_modules/expo-image/android/src/main/java/expo/modules/image/ExpoImageViewWrapper.kt
+index 75f5612..a665b38 100644
+--- a/node_modules/expo-image/android/src/main/java/expo/modules/image/ExpoImageViewWrapper.kt
```

Comment:
unrelated we need to fix the android build. we need to update to 1.8 so we can get promises from the prefetch command so we can do the preview fallback logic, but i don't want to update everything. so 1.8 was the newest that would build on ios w/ updating expo/core but this didn't work on android and i didn't notice until now. so we need to patch some tracing stuff we don't care about to make this work